src/Rabbitmq/Manager/Manager.php

```diff
@@ -21,56 +21,61 @@ class Manager {
      * Test prefix used for test mode connectionIdentifier creation.
      */
     const TEST_PREFIX = 'test_';
-    
+
     /**
-     * Internal storage for available RabbitMQ connections. 
-     * @var array 
+     * Internal storage for available RabbitMQ connections.
+     * @var array
      */
     private $connections = [];
 
     /**
      * A list of connection ids that allow for accepting incoming commands.
-     * @var array 
+     * @var array
      */
     private $sources = [];
-    
+
     /* ------------------------------------ Class Property END ----------------------------------------- */
-    
+
     /* ------------------------------------ Class Methods START ---------------------------------------- */
 
     /**
      * Add a new connection to the connection pool.
-     * 
+     *
      * @param \Maleficarum\Rabbitmq\Connection\Connection $connection
      * @param string $identifier
      * @param int $mode
      * @param int $priority
+     * @param int $publishConfirmTimeout
      * @throws \InvalidArgumentException
      * @return \Maleficarum\Rabbitmq\Manager\Manager
      */
-    public function addConnection(\Maleficarum\Rabbitmq\Connection\Connection $connection, string $identifier, int $mode, int $priority = 0) : \Maleficarum\Rabbitmq\Manager\Manager {
+    public function addConnection(\Maleficarum\Rabbitmq\Connection\Connection $connection, string $identifier, int $mode, int $priority = 0, int $publishConfirmTimeout = 0) : \Maleficarum\Rabbitmq\Manager\Manager {
```

Comment:
It always seems good to indicate the units in the name of the param, here - I don't know whether it's in seconds or milliseconds or anything else.

Reply:
This is a fair point.